setup.py

```diff
@@ -59,5 +59,9 @@
         "Documentation": "https://qiskit.org/documentation/",
         "Source Code": "https://github.com/Qiskit/red-queen",
     },
+    entry_points="""
+        [console_scripts]
+        red-queen=cli:main
```

Comment:
You need to move the file to be under the `red_queen` namespace. So `git mv cli.py red_queen/cli.py` and then change this path accordingly